include/btllib/aahash.hpp

```diff
@@ -141,6 +141,12 @@ class AAHash
   size_t get_pos() const { return pos; }
   unsigned get_hash_num() const { return hash_num; }
   unsigned get_k() const { return k; }
+  bool
+  forward() // NOLINT(readability-convert-member-functions-to-static,-warnings-as-errors)
```

Comment:
This function is redundant and removed in the latest ntHash. We can remove it here too.